packages/pluggableWidgets/switch-web/src/__tests__/Switch.spec.tsx

```diff
@@ -0,0 +1,273 @@
+import { actionValue, EditableValueBuilder, Alert, AlertProps } from "@mendix/piw-utils-internal";
+import { mount, ReactWrapper } from "enzyme";
+import { createElement } from "react";
+
+import SwitchComponent, { SwitchProps } from "../components/Switch";
+import { SwitchContainerProps } from "../../typings/SwitchProps";
+import { Switch } from "../Switch";
+
+describe("Switch", () => {
+    let switchWrapper: ReactWrapper<SwitchContainerProps, any>;
+    let switchComponent: ReactWrapper<SwitchProps, any>;
+    let switchComponentWrapper: ReactWrapper<any, any>;
+    let checkbox: ReactWrapper<any, any>;
+    let switchButtonWrapper: ReactWrapper<any, any>;
+    let switchButton: ReactWrapper<any, any>;
+    let alert: ReactWrapper<AlertProps, any>;
+    const createAndFindElements = (props: SwitchContainerProps): void => {
+        switchWrapper = mount(createElement(Switch, props));
```

Comment:
Can you convert createElement function calls to jsx syntax?

Reply:
Done, thanks.

Reply:
@Keraito if you can please review for Isa

Reply:
Addressed in https://github.com/mendix/widgets-resources/commit/1f280889d8ed4a74b64842d04f5de7d6d8e6d74d